iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/statemachine/CreateRegionGroupsProcedure.java

```diff
@@ -259,6 +275,7 @@ public void deserialize(ByteBuffer byteBuffer) {
             ThriftCommonsSerDeUtils.deserializeTRegionReplicaSet(byteBuffer);
         failedRegionReplicaSets.put(groupId, replica);
       }
+      persistPlan.deserializeForProcedure(byteBuffer);
```

Comment:
For compatibility reasons, it is important to determine if there are still redundant bytes that can be read, and if so, deserialize them

Reply:
We can ignore the compatibility requirement of Procedure. Because there must no serialized Procedures when doiong upgration

Reply:
The old confignode version without snapshot replays all Raftlogs on restart, where all procedure plans are called deserialize function. In this case, for the deserialization of newly added fields, It is a mistake not to determine whether there is margin. After deserializing, it will be replayed in configRegion, where the serialization method will be called to prevent NPE if it is not deserialized.

This is a screenshot of an error I accidentally made  in this [pr](https://github.com/apache/iotdb/pull/10743/commits/384517502743cca22fc80dad89bba4b3e1cb61b8#diff-7b1570e04d8ec47ee0731a8670ea0879d67ee8cd96d2237521f46c8ed9c39570) while trying to change something like this
![img_v2_90c4eb28-c054-474f-9d03-170e1ed8f5dg](https://github.com/apache/iotdb/assets/32640567/faa3a03b-20cd-43e0-b50f-a9ea6db9cd77)

Reply:
Fixed